app.js

```diff
@@ -63,31 +92,28 @@ app.post('/api/putLabel', (req, res) => {
   var id = req.body.id;
   var lab = req.body.label;
```

Comment:
unnecessary to declare variables when these values are only used once, use in place instead